e2e/files.spec.ts

```diff
@@ -268,11 +279,11 @@ test.describe('Files', () => {
 
           await userPage
             .getByRole('menuitem', { name: `@${validatedUserName}` })
-            .click({ timeout: testInfo.timeout * 3 })
+            .click({ timeout: defaultExpectTimeout * 3 })
```

Comment:
`testInfo.timeout` is the _test_ timeout. The expect timeout is not currently available in the `testInfo` fixture, but Playwright plans to expose the timeout in the future